test/test.robot

```diff
@@ -0,0 +1,78 @@
+*** Settings ***
```

Comment:
Is this test case meant for network security project?

Reply:
No sir it is Alcohol project sir